steely/plugins/_haskell.sh

```diff
@@ -0,0 +1,31 @@
+DIRECTORY="temporary"
```

Comment:
`tmp` is the usual?

Reply:
`usl` is the `usual`?

But for real, I don't want to encourage abbreviation. If someone else upvotes your comment, I'll bend my knee.